app/menu.js

```diff
@@ -7,18 +7,35 @@ const {getConfigDir} = require('./config');
 
 const isMac = process.platform === 'darwin';
 const appName = app.getName();
+const appVersion = app.getVersion();
 
 // based on and inspired by
 // https://github.com/sindresorhus/anatine/blob/master/menu.js
 
-module.exports = ({createWindow, updatePlugins}) => {
+module.exports = ({createWindow, updatePlugins, getPlugins}) => {
+  const showAbout = () => {
+    const pluginList = getPlugins().length === 0 ?
```

Comment:
save the plugins in a variable
`const plugins = getPlugins()`
at the top of this function, that way you avoid calling it multiple times 👍